rubix-spi/src/main/java/com/qubole/rubix/spi/fop/ObjectPoolPartition.java

```diff
@@ -174,20 +145,20 @@ public boolean decreaseObject(Poolable<T> obj)
     return true;
   }
 
-  private synchronized void objectRemoved()
+  private void objectRemoved()
   {
-    totalCount--;
+    totalCount.decrementAndGet();
   }
 
-  public synchronized int getTotalCount()
+  public int getTotalCount()
```

Comment:
should this be `getActiveConnectionCount` (the var too)?

Reply:
will change this.